### ISSUE TITLE
Migrate content from current lifeitself.org (wordpress) to new setup (Nov 2022)

### ISSUE THREAD

**Issue by rufuspollock** [life-itself/community#184]
Migrate content from current lifeitself.org (wordpress) to new setup.

## Acceptance

All content from existing wordpress instance is migrated

- [x] identified what content should be refactored e.g. deleted / turned into blog posts before migration **✅ this has been completed ready for go live, further refactoring will be carried out in phase 2 with #223**
- [X] All blog posts are at `/blog/yyyy/mm/dd/slug/`**✅ 2022-12-23 this is fixed**
- [x] All pages are migrated (that we want to migrate) and show up at their original url **✅2022-12-29 DONE and working. See spreadsheet.**
- [x] Communicate to all team, including Berlin Hub Manager (Ilja [at] lifeitself.org) who i don't think is in all-team mailing list, that switchover has happened and link to instructions for how to edit new site **✅2023-01-04 pre xmas**

### Prep

- [x] Analyse all pages
  - [x] Get a list of all pages into a spreadsheet **✅ 2022-11-07 exported Wordpress page list to [spreadsheet](https://docs.google.com/spreadsheets/d/13RVMrZ02Rw6a1BtftIUO4v7R7rXt5_phQLR4rHa343M/edit#gid=783988620)**
  - [x] Analyze them **✅2022-11-07 @rufuspollock reviewed the sheet and added actions and comments on almost all 2022-12-18 ALL DONE!**
  - [x] Clean up pages pre-migration **✅2022-12-19 we have deleted / back-blogged everything. we assigned most posts to rufus where unclear.**
- [x] Analyse blog posts and see if any we delete etc **❌2022-12-19 WONTFIX and do in #223**
  - [x] export the list 	
  - [x] see if we recategorize (how many blog posts are uncategorized) @nathenf or @EilidhRoss1 (could do this later in #223 perhaps) **❌2022-12-19 WONTFIX and do in #223**

### Migration

- [x] Migrate pages to vault/
  - [x] Remigrate with images links fixed (in frontmatter and body). **✅ 2022-12-21**
- [x] Migrate posts to vault/blog **✅ 2022-12-21**
- [x] Migrate authors / people to a people directory **✅ 2022-12-22**
- [x] Migrate categories (?) - leave for now **✅ 2022-12-22 categories have been moved over with bulk migration**
- [x] Manually migrate due to not in bulk migrate **✅ 2022-12-23 all of these are migrated now**
   - [x] https://lifeitself.org/2020/10/01/bergerac-hub-some-practicalities/ **✅ now at https://next.lifeitself.org/blog/2020/10/01/bergerac-hub-some-practicalities/**
   - [x] https://lifeitself.org/?p=22 **✅ now at next.lifeitself.org/blog/2020/02/20/contemplative-activism-event/**
   - [x] https://lifeitself.org/?p=59 **✅ now at next.lifeitself.org/blog/2020/03/22/tv-addiction/**
   - [x] https://lifeitself.org/?p=901 **✅ now at next.lifeitself.org/blog/2020/03/26/inequality-of-being/**
   - [x] https://lifeitself.org/?p=1050 **✅ now at next.lifeitself.org/blog/2020/04/25/thoughts-on-nearings-living-the-good-life/**
   - [x] https://lifeitself.org/?p=6913 **✅ now at next.lifeitself.org/blog/2022/05/25/reclaiming-spaciousness-reflections-on-conscious-food-workshop/**
   - [x] https://lifeitself.org/2018/09/17/governance-at-life-itself-previously-art-earth-tech-v1/ **✅ now at next.lifeitself.org/blog/2018/09/17/governance-at-art-earth-tech-a-proposal**
- [x] Copy changes from https://lifeitself.org/2017/12/02/our-hubs/  to new website **✅ This has been updated accordingly**

Misc

- [x] Allow `authors` frontmatter in all pages (not just blog) **✅ 2022-12-22**

### For go live

- [x] Update Navigation Bar: @catherinet1. **✅ Nav bar is now expanded on the website**
  - [x] residencies (3) - https://next.lifeitself.org/upcoming-residencies-gatherings
  - [x] about (1)
  - [x] blog (4)
  - [x] podcast (5)
  - [x] contact (6)
  - [x] people (rename The Team to People) (2)
- [x] Create central blog page (/blog) **✅2022-12-21  with the blog integration** https://github.com/life-itself/community/pull/256
- [x] Contact page (working) (/contact) **✅2022-12-21  with the blog integration**
- [x] Contact page (working) (/contact)  **✅  Tided the contact page to include basic email and social links for now. I was not able to integrate a mailchimp sign-up form, though not essential for go live**
- [x] Update main home page buttons to do what? **✅ buttons updated**
  - [x] Learn more button to go to About page **✅ links to /about**
  - [x] Join a residency button to go to Upcoming residencies page (change button text from "Join a month residency" to "Join a residency") **✅ links to /upcoming-residencies-gatherings**
  - [x] Blog page messes up on mobile **✅ 2022-12-23 fixed [2b77504](https://github.com/life-itself/community/commit/2b775045d5523cc35943c1c3879df23c732582b5)**
  - [x] Podcast page is not mobile responsive **✅ 2022-12-29 fixed in [d997fd6](https://github.com/life-itself/community/commit/d997fd677a35699b9d867e85547dd1d9e9d419bc)**
- [x] Add social media links (where) [maybe optional for launch] **❌ WONTFIX do after go live, task added to #223**

---

## Backups

Manifesto
A community for people seeking a wiser world

Though living in societies which have never been so rich, our sense of purpose and connection is often missing. A growing number of us are seeking lives that are meaningful as well as productive, yet we struggle to find a balance, finding ourselves trapped in conflict.

With Life Itself we want to offer a way to combine wiser and weller ways of living whilst remaining engaged in conventional society. We think that a radically better world is possible through the transformation of individual and collective being (or “culture”).

Our approach focuses on fostering a community of people seeking a wiser world. We do this by providing space, a network and knowledge. Space in the form of physical places that can act as homes for our community members to work, meet and stay. Network in the form of gatherings that bring the community together to reflect, learn and connect. Knowledge in the form of education and research to develop and share ideas, approaches and best practices.

Our work is underpinned by a number of foundational principles. These include: 

Primacy of Being
Love
Reverence for life
Possibility of transformation
Non-violence
The value of the mysterious
The value of reason
Deep listening
Wisdom
Non-judgement
Critical open-mindedness
Integrity
Mindfulness
Rigour and discipline
To be in community
Pragmatism
A focus on the necessity of a Big Vision

**Comment by nathenf** [comment]
@rufuspollock I moved over all pages into the [spreadsheet](https://docs.google.com/spreadsheets/d/13RVMrZ02Rw6a1BtftIUO4v7R7rXt5_phQLR4rHa343M/edit#gid=783988620). I also moved over the blog posts in a separate tab for future reference.

**Comment by rufuspollock** [comment]
@nathenf great. created task item re blog posts. Please feel free to add more tasks.

Main thing here is to understand how we reorganize pages. Secondarily how to recategorize the posts. My guess is we have taxonomy described above so we first want to get initiatives/programs list sorted as per #97

**Comment by rufuspollock** [comment]
@nathenf extensively annotated the pages list adding a sections column (with associated ["sections" sheet](https://docs.google.com/spreadsheets/d/13RVMrZ02Rw6a1BtftIUO4v7R7rXt5_phQLR4rHa343M/edit#gid=171606806) with key sections and their description). Sections are related to our info architecture discussions updated by looking at the pages.

**Comment by nathenf** [comment]
@rufuspollock I have updated this issue a little bit. I think my proposal regarding taxonomy is that we have less blog categories that are more pragmatic (Updates, Our Philosophy, Initiatives, Book Reviews, Summaries of Concepts & Traditions, Guides to Practices, Thought Pieces) and then use tags for specific initiatives and themes. 

I don't know for sure if this is the best way to go, but what I see right now RE current categories doesn't seem useful.

**Comment by rufuspollock** [comment]
@nathenf (cc @khalilcodes) let's do things in stages:

1. First pass review of content to check if anything to delete / drop
2. Do the content migration (automated) <-- at this point new site can go live
3. Refactor and tidying - this can happen later (once new site 

Things like taxonomy stuff are part of item (3) and can happen later. I actually suggest we even move that to a new issue #223 (just created).

@nathenf right now the priority is to finish the review of pages and delete pages (in wordpress that we don't want - backing up if we really need). Can you prioritize that.

@khalilcodes we need to do the migration of content once that is done. Also for clarity i think we want almost all of that content to end up in the vault (? - we should briefly sync about this).

I'm going to try and update this issue to reflect this.

**Comment by rufuspollock** [comment]
FIXED. Small remaining bugs can be done post-go-live and have been moved into #170
